Updated index.html

OGGY418 opened the pull request:
Refactor: Restructure index.html meta tags (resolves #28)

OGGY418 committed:
Updated index.html

Refactor: Restructure index.html meta tags (resolves #28)

Anshgrover23 commented on index.html:
```diff
@@ -3,28 +3,31 @@
     <meta property="og:type" content="website" />
+    <meta property="og:title" content="Ansh's Portfolio" />
+    <meta property="og:description" content="Ansh Grover's personal portfolio website." />
+    <meta property="og:image" content="https://anshgrover23.github.io/anshs-portfolio/assets/images/my-portfolio-preview.jpg" />
```
this does not work.

Anshgrover23 commented on index.html:
```diff
@@ -3,28 +3,31 @@
+    <meta name="twitter:url" content="https://anshgrover23.github.io/anshs-portfolio" />
+    <meta name="twitter:title" content="Ansh's Portfolio" />
+    <meta name="twitter:description" content="Ansh Grover's personal portfolio website." />
+    <meta name="twitter:image" content="https://anshgrover23.github.io/anshs-portfolio/assets/images/my-portfolio-preview.jpg" />
```
we don't use github pages.

OGGY418 replied:
please check i have resolved and updated

OGGY418 committed:
Fix: Updated OG and Twitter URLs

This pull request updates the Open Graph and Twitter meta tags in index.html to use the correct live domain (anshgrover.me).

This change fixes the issue where the social media previews were not working because the URLs were incorrectly pointing to the GitHub Pages domain. This resolves the problems mentioned in the previous pull request's review.
